server/prisma/schema.prisma

```diff
@@ -8,28 +8,109 @@ datasource db {
   url      = "file:./dev.db"
 }
 
-// This comes with the boilerplate.
 model User {
-  id                  String          @id @default(cuid())
-  createdAt                        DateTime        @default(now())
-  updatedAt                        DateTime        @updatedAt
-  email           String  @unique
-  password      String
-  role          String
-  lastLogin     DateTime?  
-  // LL: https://www.smashingmagazine.com/2017/11/safe-password-resets-with-json-web-tokens/
-  resetPasswordToken String
-  dateResetPasswordRequest DateTime?
-  validateEmailToken String
-  isEmailValidated Boolean
-  name  String
+  id                        Int                         @id @default(autoincrement())
+  firstName                 String
+  lastName                  String
+  email                     String                      @unique
+  feed                      Feed[]
+  board                     Board[]
+  post                      Post[]
+  userSourceFavorite        UserSourceFavorite[]
+  createdAt                 DateTime                    @default(now())
+  setting                   Setting?
+  userActivatedSubscription UserActivatedSubscription[]
+
+  @@map("users")
 }
 
-model Feed {
+// user settings like theme
+
+model Setting {
+  id     Int    @id @default(autoincrement())
+  user   User   @relation(fields: [userId], references: [id])
+  userId Int    @unique
+  theme  String
+
+  @@map("settings")
+}
+
+// integrations
+
+model Subscription {
```

Comment:
SubscriptionPlanInfo
SubscriptionSpec

Reply:
if you have something called Subscription and UserActivatedSubscription you 100% need comments above them